BridgeAppSDK/SBAActivityTableViewController.swift

```diff
@@ -163,8 +175,12 @@ public class SBAActivityTableViewController: UITableViewController, SBASharedInf
             where shouldRecordResult(schedule, taskViewController: taskViewController) {
             
             updateScheduledActivity(schedule, taskViewController: taskViewController)
+            taskViewController.task?.updateTrackedDataStores(true)
```

Comment:
not obvious what the parameter does until you look at the function definition--could you include the parameter name in the call?